lib/nodejs/reflector.js

```diff
@@ -323,4 +344,75 @@ function OnConnection( socket ) {
     } );
 }
 
-exports.OnConnection = OnConnection;
\ No newline at end of file
+function Evaluate( namespace, node, expression ) {
+
+    return new Promise( function( resolve, reject ) {
+
+        var firstclient = Object.keys( global.instances[ namespace ].clients )[ 0 ];
+        firstclient = global.instances[ namespace ].clients[ firstclient ];
```

Comment:
Clever reuse of the `firstclient` object here, but I don't love it.  I think such a thing can get confusing enough that for the sake of others looking at the code I would recommend creating two variables, the first of which could be named `firstClientName` or some such.